File: Aula 9/Aula.java


public class HelloWorld {

     public int abs_if_else(int valor){
           if (valor < 0) {
            return -valor;
           } else {
            return valor;
           }
        }


     public int abs_ternario(int valor){
      return (valor < 0) ? -valor : valor; 

    }
    
    public int abs_somatorio_for(int valor){
        int soma = 0;
        for (int i = 0; i <= valor; i++) {
            soma += i;
        }
        return soma;
    }

    public int abs_somatorio_while(int valor){
        int soma = 0;
        int i = 0;
        while (i <= valor){
            soma += i;
            i += 1;
        }
    return soma;
    }

    public int somatorio_do_while(int valor){
        int soma = 0;
        int i = 0; 
    }
    
    
    
    
    

File: Aula 9/Aula.py
def abs_if_else(valor):
    if valor < 0:
        return -valor
    else:
        return valor

def abs_ternario(valor):
    return -valor if valor < 0 else valor 
   
def somatorio_for(n):
    soma = 0
    for i in range(n + 1):
        soma += i
        return soma

def somatorio_while(n):
    soma = 0 
    i = 0
    while i <= n:
        soma += i 
        i += 1
        return soma 

def somatorio_do_while(n):
    soma = 0 
    i = 0
    while True:
        soma += i 
        i += 1
        if i > n:
            break
    return soma

print("Valor absoluto (if-else):", abs_if_else(-10))
print("Valor absoluto (ternário)", abs_ternario(-10))
print("Somatório (for)", somatorio_for(10))
print("Somatório (while)", somatorio_while(10))
print("Somatório (do-while)", somatorio_do_while(10))
